### PR TITLE
Fdohru2

### PR DESCRIPTION
**Pull request by Feridddd**

### CODE REVIEW

File: src/main/java/Menu/Main.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package Menu;
```
Comment (olobanok) [inline src/main/java/Menu/Main.java]: package names should be in lowercase

File: src/main/java/hotel/Hotel.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                System.out.println("Room corresponds to your criteria");
+                break;
+            } else {
+                System.out.println("There is no rooms that matches your criteria");
+            }
+        }
+    }
+    public void isCorrespondAll() {
+        for (Room r : listRooms) {
+            if (r.getNameType() == "Budget") {
```
Comment (olobanok) [inline src/main/java/hotel/Hotel.java]: do not compare strings using ==

File: src/main/java/Menu/Main.java (hunk 1 above)
Comment (Feridddd) [reply src/main/java/Menu/Main.java]: Done

File: src/main/java/hotel/Hotel.java (hunk 2 above)
Comment (Feridddd) [reply src/main/java/hotel/Hotel.java]: Done

**Pull request merged**